plenigo_plugin/PlenigoShortcodeManager.php

```diff
@@ -846,74 +864,85 @@ public function buildCheckoutSnippet($atts, $tag, $prodId) {
         $checkoutSnippet = "alert('The button was not configured correctly')";
 
         if ($prodId !== "") {
+
             // creating atts plenigo-managed product
             if ($tag === 'pl_failed') {
                 $product = \plenigo\models\ProductBase::buildFailedPaymentProduct();
             } else {
+                $title = null;
+                if ($usePostTitle === "1") {
+                    $title = get_the_title($postId);
+                }
+
                 if (!empty($price)) {
                     if (empty($quantity) || !is_numeric($quantity)) {
                         $quantity = 1;
                     }
-                    $title = null;
-                    if ($quantity > 1) {
+                    $productTitle = $title;
+                    if (empty($productTitle)) {
                         $productData = null;
                         try {
                             $productData = \plenigo\services\ProductService::getProductData($prodId);
+                            if ($productData !== null && !empty($productData->getTitle())) {
+                                $title = $productData->getTitle();
+                            } else {
+                                $productTitle = "Item";
+                            }
                         } catch (\Exception $exc) {
                             plenigo_log_message($exc->getMessage() . ': ' . $exc->getTraceAsString(), E_USER_WARNING);
                         }
-                        if ($productData !== null && !empty($productData->getTitle())) {
-                            $title = "$quantity x " . $productData->getTitle();
-                        } else {
-                            $title = "$quantity x Item";
-                        }
-                    }
 
+                    }
+                    if ($quantity > 1) {
+                        $title = "$quantity x " . $productTitle;
```

Comment:
"$quantity x "?

Reply:
Yes they wanted the format of "4 x ItemTitle" in the bill

Reply:
why not 
$title = "$quantity x $productTitle";
or
$title = .$quantity . ' x ' . $productTitle;